srcpkgs/xmobar/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'xmobar'
+pkgname=xmobar
+version=0.27
+revision=1
+build_style=haskell-stack
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel
+ $(vopt_if xft libXft-devel)
+ $(vopt_if iwlib wireless_tools-devel)
+ $(vopt_if xpm libXpm-devel)
+ $(vopt_if dbus libxml2-devel $(vopt_if mpris libxml2-devel))
+ $(vopt_if alsa alsa-lib-devel)"
+short_desc="A Minimalistic Text Based Status Bar"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="BSD-3-Clause"
+homepage="http://xmobar.org"
+distfiles="https://hackage.haskell.org/package/${pkgname}-${version}/${pkgname}-${version}.tar.gz"
+
+build_options="dbus threaded utf8 xft mpd mpris inotify iwlib alsa datezone
+ xpm uvmeter weather conduit"
+build_options_default="threaded utf8 xft inotify iwlib datezone weather xpm"
+
+nopie_files="/usr/bin/xmobar"
+
+nocross=yes
+
+vopt_hflag() {
+	echo $(vopt_if $1 "--flag ${pkgname}:with_$1" "--flag ${pkgname}:-with_$1")
+}
+
+mk_string_build_args() {
+	for optn in $1
+	do
+		echo -n " $(vopt_hflag $optn)"
+	done
+}
+
+make_build_args=$(mk_string_build_args "$build_options")
+
+post_install() {
+	vlicense license
+}
+
+
+checksum=74748670b9bcfc3e6f8844c1ba86fc91bfac1bd27e0cecf6d87569834122fd29
```

Comment:
should be directly below distfiles=

Reply:
Done. This kind of things should be signaled by xlint, no ?

Reply:
xlint doesn't have a rule for that yet. Maybe it will be extended sometimes to include some of the not yet official (but used everywhere) style rules.